CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [6.1.0]
+### Changed
+- Update BIP-39 implementation from `@scure/bip39` to MetaMask fork version `@metamask/scure-bip39` ([#101](https://github.com/MetaMask/key-tree/pull/101))
```

Comment:
I know that it's given in the PR, but maybe it's worth explaining what the fork does over the original implementation?

Reply:
sure can do!

Reply:
Done [here](https://github.com/MetaMask/key-tree/pull/105/commits/36bc2df2abc25c049a5823db86848a4295b4d1d3)